packages/business-rules/src/schemas/full-appeal/insert.js

```diff
@@ -115,6 +116,14 @@ const insert = pinsYup
               }
               return pinsYup.string().nullable();
             }),
+            tellingTheLandowners: pinsYup
+              .array()
+              .nullable()
+              .test('allTellingTheLandowners', buildTripleConfirmMessage, validateTripleConfirm),
```

Comment:
If the triple confirm validator was created as a custom yup method then these wouldn't need to be imported and it would make this validator cleaner and easier to re-use for this appeal type and also future appeal types should we need to.

The validator could be put in `/src/validators/common` as it's not specific to a page.

An example of a custom validator is `packages/business-rules/src/validators/common/conditional-text.js`
An example of adding the custom validator to yup is `packages/business-rules/src/lib/pins-yup.js`